db-engine/src/main/java/com/splicemachine/db/iapi/sql/conn/LanguageConnectionContext.java

```diff
@@ -452,6 +452,9 @@ Similar to internalCommit() but has logic for an unsynchronized commit
      */
 	String getCurrentUserId(Activation a);
 
+	void setCurrentUser(Activation a, String userName);
+
+	void setCurrentGroupUser(Activation a, List<String> groupUsers);
```

Comment:
`setCurrentGroupUser` and `groupUsers` do not seem consistent and it's not clear to me what it sets - the users that belong to a group or the groups a user belongs to or something else. Please add a javadoc and also make the names consistent.